website/docs/install-config/install/aws/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: AWS installation
+---
+
+You can install authentik to run on AWS with a Cloudformation template.
+
+### Prerequisites
+
+- An AWS account.
+- An [AWS Certificate Manager](https://aws.amazon.com/certificate-manager/) certificate. Take note of the ARN of the certificate.
+
+### Installation
+
+Create a Cloudformation stack [with our template](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?stackName=authentik&templateURL=https://authentik-cloudformation-templates.s3.amazonaws.com/authentik.ecs.latest.yaml).
```

Comment:
maybe a note that you already need to be logged into AWS to access this template?